src/Sentry.Profiling/ProfilingIntegration.cs

```diff
@@ -10,6 +10,8 @@ public class ProfilingIntegration : ISdkIntegration
     /// <inheritdoc/>
     public void Register(IHub hub, SentryOptions options)
     {
-        options.TransactionProfilerFactory = SamplingTransactionProfilerFactory.Create(options);
+        if (options.IsProfilingEnabled) {
```

Comment:
I'll buy 10 beers to the first person who adds something like `dotnet format` that works in this repo :)

```suggestion
        if (options.IsProfilingEnabled)
        {
```